atomicfu/src/commonMain/kotlin/kotlinx/atomicfu/Trace.common.kt

```diff
@@ -60,12 +60,20 @@ public expect fun TraceBase.named(name: String): TraceBase
 public expect val traceFormatDefault: TraceFormat
 
 public open class TraceBase internal constructor() {
-    @JsName("atomicfu\$Trace\$append\$")
-    @PublishedApi
-    internal open fun append(text: String) {}
+    @JsName("Trace\$append\$1\$atomicfu\$")
+    public open fun append(event: Any) {}
```

Comment:
Please fix all the implementors of this class, because they are still using `text` parameter name